Integrations/integration-Alexa.yml

```diff
@@ -0,0 +1,119 @@
+description: Alexa provides website ranking information that can be useful in determining
+  if the domain in question has a strong web presence.
+detaileddescription: To use this integration, simply call ```!awis-info```. !awis
+  accepts "url" as a parameter to query.
+configuration:
+- display: Sensitivity threshold for configuring which domains are suspicious versus
```

Comment:
Maybe a link to an explanation about the threshold?

Reply:
This will be addressed in documentation.